authors/apps/authentication/tests/base.py

```diff
@@ -0,0 +1,22 @@
+"""Base module for all external test cases"""
+
+from rest_framework.test import APITestCase, APIClient
+from ..models import User
+from ..serializers import RegistrationSerializer
+
+
+class BaseTest(APITestCase):
```

Comment:
@BarnaTB Please rebase from the current develop so as you get the current changes and maintain how the tests were written.